tests/general/KeywordBearTest.py

```diff
@@ -214,3 +214,22 @@ def test_keyword_regex(self):
             self.assertEqual(result[0].message, 'The line contains the keyword'
                                                 " 'Issue #123' which resulted "
                                                 'in a match with given regex.')
+
+    def test_wrong_language(self):
+        self.section.append(Setting('language', 'anything'))
+        annotation_bear_result_type = namedtuple('result', 'contents')
+        dep_results = {
+            'AnnotationBear': [
+                annotation_bear_result_type(
+                  'coalang specification for anything not found.'
+                  )
+            ]
+        }
+
+        text = ['# todo 123']
+
+        with execute_bear(self.uut, 'F', text,
```

Comment:
Please use named arguments - they're easier to read. Every other test uses them and consistent is better :)

See line 211 of this file.